src/WingetCreateCLI/WingetCreateCLI.csproj

```diff
@@ -77,6 +77,19 @@
       <Generator>PublicResXFileCodeGenerator</Generator>
     </EmbeddedResource>
   </ItemGroup>
+    
+  <!--Include VCLib binary dependencies-->  
+  <ItemGroup Condition="'$(SelfContained)' == true">
```

Comment:
Will this fail when built outside the release pipeline?

Reply:
I didn't have any issues building this outside of the release. Either in vs or using dotnet publish in terminal with or without those files present.